codalab/apps/newsletter/scripts/send_emails_to_mailchimp.py

```diff
@@ -0,0 +1,27 @@
+import json
+import requests
+
+from django.contrib.auth import get_user_model
+from codalab import settings
+
+User = get_user_model()
+
+
+def run():
+    for user in User.objects.all():
+        data = {
+            "email_address": user.email,
+            "status": "subscribed",
+        }
+
+        requests.patch(
+            settings.MAILCHIMP_MEMBERS_ENDPOINT_ALL,
+            auth=("", settings.MAILCHIMP_API_KEY),
+            data=json.dumps(data)
+        )
+
+        requests.post(
```

Comment:
why do we patch and then post the same data?

Reply:
Mailchimp suggests this in their documentation. First it checks if the user is in their API and if not, we'll add them. Not ideal but that's how they've got it set up

Reply:
So, should we only `post` if `patch` returns a non-ok response?

Reply:
Fixed in latest push